README.md

```diff
@@ -25,151 +22,125 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Metrics Webapp <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> [Metric WebApp] is a mobile web application that allows users to view a list of countries. The wepapp has two pages: the home page and the details page. The home page displays a list of countries that could be filtered by country name. When the user clicks a specific country, the application navigates to the details page where it allows the user to view individual country with details like it's coat of arms, language and many more..
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="built-with">REACT-REDUX</a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-<!-- Features -->
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JAVASCRIPT</a></li>
+    <li><a href="https://getbootstrap.com/docs/5.3/getting-started/introduction/">BOOTSTRAP</a></li>
+    <li><a href="https://webpack.js.org/guides/getting-started/">WEBPACK</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">GITFLOW</a></li>
+    <li><a href="https://reactrouter.com/en/main/start/tutorial#setup">REACT ROUTER DOM V6</a></li>
+    <li><a href="https://react.dev/learn">REACTJS</a></li>
+    <li><a href="https://redux-toolkit.js.org/">REDUX TOOLKIT</a></li>
+    <li><a href="https://jestjs.io/docs/getting-started">JEST</a></li>
+  </ul>
+</details>
 
-### Key Features <a name="key-features"></a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-> Describe between 1-3 key features of the application.
+<!-- Key Features -->
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+### Key Features <a name="key-features"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+-[Home page that will display list of item] -[Details page for detailed item] -[Back button to navigate to the home page]
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🔭 Live Demo <a name="future-features"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link]()
+[Live demo](react-capstone-metrics-app.netlify.app)
```

Comment:
- [ Required ] I think it would be nice if you deploy this application using any of the following platforms Netlify, GitHub pages, or Render as stated in the project requirements. Currently, when I click on the link I get the error below.

## Error when visiting deployed application
![Screenshot from 2023-11-24 22-11-06](https://github.com/Umuhan/react-capstone-metrics-app/assets/16392046/9b90faf6-6cf4-494b-9682-c20efe6f68d8)

Reply:
Thanks. Done